spec/dealership_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rspec'
+require './lib/car'
+require './lib/dealership'
+require 'pry'
+
+RSpec.describe Dealership do
+  before(:each) do
+    @car_1 = Car.new("Ford Mustang", 1500, 36)
+    @car_2 = Car.new('Toyota Prius', 1000, 48)
+    @car_3 = Car.new("Toyota Tercel", 500, 48)
+    @car_4 = Car.new("Chevrolet Bronco", 1250, 24)
+
+    @dealership = Dealership.new('Acme Auto', '123 Main Street')
+  end
+
+  it 'dealerships initially have no inventory' do
+    expect(@dealership.inventory).to eq([])
+    expect(@dealership.inventory_count).to eq(0)
+  end
+
+  it 'can add cars to inventory' do
+    @dealership.add_car(@car_1)
+    @dealership.add_car(@car_2)
+    expect(@dealership.inventory_count).to eq(2)
```

Comment:
You would also need to add an expect for `.inventory` in this test as well. The interaction pattern is expecting an array of the car objects that have been added. 
```
pry(main)> dealership.inventory
#=> [#<Car:0x00007fa53b9ca0a8...>, #<Car:0x00007fccd2985f48...>]
```